### PR TITLE
RavenDB-17053 - Can_use_cluster_tx_mixed_with_tx fails

### PR DESCRIPTION
**Pull request by shaharhikri**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17053/SlowTests.Issues.RavenDB12601.Canuseclustertxmixedwithtx

### Additional description

Can_use_cluster_tx_mixed_with_tx  fails (could not dispose server..).

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- Tests have been added that prove the fix is effective or that the feature works

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

File: test/SlowTests/Issues/RavenDB_12601.cs [hunk 1]
```diff
@@ -160,10 +167,12 @@ public async Task Can_use_cluster_tx_mixed_with_tx()
 
                     session.Store(user);
                     session.SaveChanges();
-
+                    
+                    trxn += GetNumOfUpdateLicenseLimitsCommand(Server); // could be 1 or more 'UpdateLicenseLimitsCommand' before the 'ClusterTransactionCommand'
```
Comment (karmeli87) [inline test/SlowTests/Issues/RavenDB_12601.cs]: So this is still a racy.
Better to wait for the expected number of `UpdateLicenseLimitsCommand`.

File: test/SlowTests/Issues/RavenDB_12601.cs [hunk 2]
```diff
@@ -233,6 +242,21 @@ public async Task Can_use_cluster_tx_mixed_with_tx()
...
+
+        private string GetLogs(RavenServer n)
+        {
+            using (n.ServerStore.Engine.ContextPool.AllocateOperationContext(out ClusterOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                return Cluster.CollectLogs(context, n);
+            }
+        }
+
```
Comment (karmeli87) [inline test/SlowTests/Issues/RavenDB_12601.cs]: let's use `LastRaftIndexForCommand` instead of those

**Review by karmeli87 — CHANGES_REQUESTED** [review]

File: test/SlowTests/Issues/RavenDB_12601.cs (hunk 1 above)
Comment (karmeli87) [reply test/SlowTests/Issues/RavenDB_12601.cs]: or for the expected raft index

**Review by ppekrol — APPROVED** [review]

**Review by karmeli87 — APPROVED** [review]

File: test/SlowTests/Issues/RavenDB_12601.cs (hunk 2 above)
Comment (shaharhikri) [reply test/SlowTests/Issues/RavenDB_12601.cs]: Done

File: test/SlowTests/Issues/RavenDB_12601.cs (hunk 1 above)
Comment (shaharhikri) [reply test/SlowTests/Issues/RavenDB_12601.cs]: Done

**Pull request merged**